docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -74,33 +74,33 @@ Setting up your environment to build documentation locally
 
 To build documentation locally, ensure you have a working :ref:`development environment <environment_setup>`.
 
-To work with documentation on your local machine, you need to have python-3.5 or greater and the
-following packages installed:
+To work with documentation on your local machine, you need to have python-3.5 or greater and install the `Ansible dependencies`_ and `documentation dependencies`_, which are listed in two :file:`requirements.txt` files to make installation easier:
 
-    - ``gcc``
-    - ``jinja2``
-    - ``libyaml``
-    - ``make``
-    - ``Pygments``
-    - ``pyparsing``
-    - ``PyYAML``
-    - ``rstcheck``
-    - ``six``
-    - ``sphinx``
-    - ``sphinx-notfound-page``
-    - ``straight.plugin``
-
-These required packages are listed in two :file:`requirements.txt` files to make installation easier:
+.. _Ansible dependencies: https://github.com/ansible/ansible/blob/devel/requirements.txt
+.. _documentation dependencies: https://github.com/ansible/ansible/blob/devel/docs/docsite/requirements.txt
 
 .. code-block:: bash
 
     pip install --user -r requirements.txt
     pip install --user -r docs/docsite/requirements.txt
 
-You can drop ``--user`` if you have set up a virtual environment (venv/virtenv).
+The :file:`docs/docsite/requirements.txt` file allows a wide range of versions and may install new releases of required packages. New releases of these packages may cause problems with the Ansible docs build. If you want to install tested versions of these dependencies, use :file:`docs/docsite/known_good_reqs.txt` instead:
+
+.. code-block:: bash
+
+    pip install --user -r requirements.txt
+    pip install --user -r docs/docsite/known_good_reqs.txt
```

Comment:
@acozine @samccann I missed this but it should be something like `pip install -r docs/docsite/requirements.txt -c docs/docsite/known_good_reqs.txt`. Which basically constraints what should be installed.